README.rst

```diff
@@ -412,6 +414,7 @@ An example ``pyproject.toml`` configuration file:
    line-length = 80                  # Passed to isort and Black, override their config
    target-version = ["py312"]        # Passed to Black, overriding its config
    log_level = "INFO"
+   preview = false                   # Passed to Black, overriding its config
```

Comment:
For simplicity, we could disallow `preview =` in Darker configuration files. The user can configure it in `[tool.black]` anyway. I've done the same for `skip-magic-trailing-comma` and `skip-string-normalization` earlier.

Reply:
I invited you @tehunter as a collaborator on the Darker repository. Feel free to review #607 which removes two options from `[tool.darker]` but keeps them as command line options and as options in `[tool.black]`.